addon/helpers/chunk.js

```diff
@@ -31,22 +26,6 @@ export function chunk(num, array) {
   }
 }
 
-export default Helper.extend({
```

Comment:
So effectively, no more stateful helpers b/c of the way glimmer templates work?  Stateful was necessary b/c....updated values passed to helpers were not invalidated?

Reply:
Yes, that was true for the arrays and objects. Helpers are now recomputed when they change